### PR TITLE
Updates SAML settings and service provider documention for reseller s…

### PR DESCRIPTION
**Pull request by m-orfanos**
#### Issue

SAML settings and service providers documentation needs to be updated to take into account the reseller scoping

#### Solution

Do it

#### Related PRs

- Core: https://github.com/cloudops/cloudmc-core/pull/1030
- UI: https://github.com/cloudops/cloudmc-ui/pull/1145

### CODE REVIEW

File: source/includes/administration/_saml_settings.md [hunk 1]
```diff
@@ -6,13 +6,18 @@ Allows to change the configuration for the service provider integrated with SAML
 
 #### List SAML settings
 
-`GET /saml_settings`
+`GET /saml_settings?organization_id=:organization_id`
 
 ```shell
 # Retrieve SAML settings
-curl "https://cloudmc_endpoint/api/v1/saml_settings" \
+curl "https://cloudmc_endpoint/api/v2/saml_settings?organization_id=:organization_id" \
```
Comment (ehudon) [inline source/includes/administration/_saml_settings.md]: Do we keep both endpoints or only v2?

File: source/includes/administration/_saml_settings.md [hunk 2]
```diff
@@ -85,24 +146,49 @@ curl -X POST "https://cloudmc_endpoint/rest/saml_settings/:id" \
...
 Updates a specific SAML settings
 
-Required | &nbsp;
----------- | -----------
-`certificate`<br/>*string* | The certificate send in the saml response for the service provider to validate the signature. It must use the SHA-256 algorithm.
-`privateKey`<br/>*string* | The private key used to sign the SAML request. It must be an RSA key of 2048 bits
+| Required                                                           | &nbsp;                                                                                                                           |
+| ------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------- |
+| `id`<br/>_UUID_                                                    | The id of the SAML setting.                                                                                                      |
+| `certificate`<br/>_string_                                         | The certificate send in the SAML response for the service provider to validate the signature. It must use the SHA-256 algorithm. |
```
Comment (ehudon) [inline source/includes/administration/_saml_settings.md]: `sent`?

File: source/includes/administration/_saml_settings.md (hunk 1 above)
Comment (m-orfanos) [reply source/includes/administration/_saml_settings.md]: TBH I'm not sure. @jasondias9 would you happen to know?

Comment (jasondias9) [reply source/includes/administration/_saml_settings.md]: For this doc should just be v2 I believe. @m-orfanos I think your base branch should be `cmc-dev-v2`

File: source/includes/administration/_saml_settings.md (hunk 2 above)
Comment (m-orfanos) [reply source/includes/administration/_saml_settings.md]: 5857a48f1c1cef505dff25e6c7a641a21f551bb8

**Review by ehudon — APPROVED** [review]
LGTM 🚀

**Review by aaronalmeida — APPROVED** [review]
LGTM! 👍

File: source/includes/administration/_saml_settings.md (hunk 1 above)
Comment (m-orfanos) [reply source/includes/administration/_saml_settings.md]: I tested on v1, all looks good 👍

Comment (m-orfanos) [reply source/includes/administration/_saml_settings.md]: We should keep it in v1 imo

Comment (jasondias9) [reply source/includes/administration/_saml_settings.md]: Yeah looks like thats the move. Sorry for the confusion.

**Pull request merged**